packages/reference/src/common/ReferenceEditor.tsx

```diff
@@ -2,22 +2,21 @@ import * as React from 'react';
 import deepEqual from 'deep-equal';
 import { FieldConnector } from '@contentful/field-editor-shared';
 import { EntityProvider } from './EntityStore';
-import { ViewType, FieldExtensionSDK, Action } from '../types';
+import { ViewType, FieldExtensionSDK, Action, Entry, ContentType } from '../types';
+
+// TODO: Rename common base for reference/media editors to something neutral,
+//  e.g. `LinkEditor<T>`.
 
 export interface ReferenceEditorProps {
   /**
-   * is the field disabled initially
+   * Whether or not the field should be disabled initially.
    */
   isInitiallyDisabled: boolean;
-
   sdk: FieldExtensionSDK;
-
   viewType: ViewType;
-
+  renderCard?: (props: CustomEntryCardProps) => React.ReactElement | false;
```

Comment:
Might make sense to rename `renderCard` to `customEntryCardElement` - feels a bit more descriptive as a prop. What do you think @DanweDE @suevalov ?

Reply:
this is a function returning an element so the card prop should include a verb I think.

Reply:
Settling for `renderCustomCard`